Firebase/Auth/Source/FIRAuth.m

```diff
@@ -121,6 +121,18 @@
  */
 static NSString *const kMissingPasswordReason = @"Missing Password";
 
+/** @var kVerifyPhoneNumberReauthOperation
+    @brief String passed to the backend to indicate that the current Phone Number Auth flow is
+        initiated by a reauthentication operation.
+ */
+static NSString *const kVerifyPhoneNumberReauthOperation = @"REAUTH";
```

Comment:
These parameters aren't phone-auth specific. I'm more inclined to put them into one place, say, use a enum and function/method to convert to strings, or we can simply make a list of strings somewhere for each of the constant values:
https://cs.corp.google.com/piper///depot/google3/googledata/apiserving/config/identitytoolkit/relyingparty/proto/api_enums.proto?dr&rcl=169764806&l=50

We'll need this for the reauthenticate operate, too.

Reply:
Went the Enum + static function route.